packages/angular/cli/src/utilities/package-metadata.ts

```diff
@@ -292,19 +292,12 @@ export async function getNpmPackageJson(
   const { usingYarn = false, verbose = false, registry } = options;
   ensureNpmrc(logger, usingYarn, verbose);
   const { packument } = await import('pacote');
-  const resultPromise = packument(packageName, {
+  const response = packument(packageName, {
     fullMetadata: true,
     ...npmrc,
     ...(registry ? { registry } : {}),
   });
 
-  // TODO: find some way to test this
```

Comment:
Not sure why this was here @clydin maybe you remember? This needs to be removed as otherwise certain errors are not fatal. Example: https://app.circleci.com/pipelines/github/angular/angular-cli/26850/workflows/10782a5e-36c3-4de6-ae85-0ebcfe456e4e/jobs/356140/parallel-runs/0/steps/0-105

The test previously failed because when trying to download the temp version of the CLI. This check is bypassed since pacote is not used.

Reply:
May have been leftover from before the package names were validated to be registry names.